srcpkgs/linux5.15/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'linux5.15'
 pkgname=linux5.15
 version=5.15.11
-revision=1
```

Comment:
Ah sorry, I had missed this. We don't increase the revision for changes like this, a new kernel revision will come out soon enough.

@leahneukirchen do you think this is ok?

Reply:
Alright, unless there is an objection, I'll fix that tomorrow.

Reply:
Fixed.